### PR TITLE
Add Valen back

### PR DESCRIPTION
**Pull request by TotoR115**
Enhanced Valen encounter : 
- Valen will now only show up if "Robed Figure" is dead
- New name for f_Valen -> Vampire
- Correction of Class and Race of f_vamp.cre

### CODE REVIEW

File: drizztsaga/drizztsaga.tp2 [hunk 1]
```diff
@@ -355,19 +359,47 @@ ACTION_IF GAME_IS ~BGT~ BEGIN
...
 END
 
+
 ///////////////////////////////////////////////////////////////////////////////////////////////
-//// New encounter: Valen in the Undercity - an optional component 			  /////////////////
+//// Nerf items: items are now "only" +3 and undroppable - an optional component/////////////////
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-BEGIN @1509
+BEGIN @1510
```
Comment (Roberciiik) [inline drizztsaga/drizztsaga.tp2:367]: @1509 tra number can be reused as the component was removed.

File: drizztsaga/drizztsaga.tp2 [hunk 2]
```diff
@@ -355,19 +359,47 @@ ACTION_IF GAME_IS ~BGT~ BEGIN
 	BUT_ONLY
 END
 
+
 ///////////////////////////////////////////////////////////////////////////////////////////////
-//// New encounter: Valen in the Undercity - an optional component 			  /////////////////
+//// Nerf items: items are now "only" +3 and undroppable - an optional component/////////////////
```
Comment (Roberciiik) [inline drizztsaga/drizztsaga.tp2:364]: The change seems to be unrelated to the current Pull Request. Could you please revert the item nerfing and leave only Valen-related changes? Thanks.

Comment (TotoR115) [reply drizztsaga/drizztsaga.tp2:364]: yes, that is a mistake

**Comment by TotoR115** [comment]
Now it should be good: no more unrelated changes.

**Comment by ALIENQuake** [comment]
@TotoR115 
Many thanks for your contribution!
